src/components/axes/categoryAxis.ts

```diff
@@ -121,11 +155,11 @@ export module Axes {
         var height = self._isHorizontal() ? axisHeight - self._maxLabelTickLength() - self.tickLabelPadding() : bandWidth;
         var writeOptions = {
           selection: d3.select(this),
-          xAlign: xAlign[self.orient()],
-          yAlign: yAlign[self.orient()],
-          textRotation: self.tickLabelAngle()
+          xAlign: xAlign[self.orientation()],
+          yAlign: yAlign[self.orientation()],
+          textRotation: self._tickLabelAngle
```

Comment:
Keep to the original implementation of self.tickLabelAngle() unless you have a strong opinion against